src/common/command_buffers/details/clear.hpp

```diff
@@ -169,4 +169,164 @@ namespace commandbuffers
   public:
     int stencil;
   };
+
+  class ClearBufferfvCommandBufferRequest final
```

Comment:
These new classes could be inherited from a base class such as `ClearBufferBaseRequest`